manticore/platforms/platform.py

```diff
@@ -43,14 +43,6 @@ def __init__(self, idx, name):
         super().__init__(msg)
 
 
-class ConcretizeSyscallArgument(OSException):
```

Comment:
Not used?

Reply:
Correct. Probably forgotten during a refactor or something